src/common/constants.ts

```diff
@@ -25,6 +25,8 @@ export const domains = [
     'deriv-com-git-fork',
 ] //deriv-com-git-fork for vercel server, localhost - for developer mode
 
+export const branding_domains = [...domains, 'binary.sx']
```

Comment:
Please set the string in a variable and use it here

Reply:
okay